hypervisor/network.go

```diff
@@ -216,18 +308,18 @@ func (nc *NetworkContext) netdevInsertFailed(idx int, name string) {
 
 func (nc *NetworkContext) configureInterface(index, pciAddr int, name string, inf *api.InterfaceDescription, result chan<- VmEvent) {
 	if inf.TapName == "" {
-		inf.TapName = network.NicName(nc.sandbox.Id, index)
```

Comment:
If you want to change the tapname, please update network.NicName function, other place rely on this naming rule.

Reply:
Sure